.github/workflows/build-images-ci.yaml

```diff
@@ -244,6 +244,50 @@ jobs:
             NOSTRIP=1
             OPERATOR_VARIANT=${{ matrix.name }}
 
+      - name: Install Bom and generate SBOM
+        shell: bash
+        run: | 
+          curl -L https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz -o bom 
+          tar -xvf bom
+          ./bom generate -o sbom_source_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --dirs=.
+          ./bom generate -o sbom_image_ci_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}
+          ./bom generate -o sbom_image_ci_race_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-race 
+          ./bom generate -o sbom_image_ci_unstripped_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
+          --image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/${{ matrix.name }}-ci:${{ steps.tag.outputs.tag }}-unstripped 
+          
+      - name: Download artifact for source SBOM
+        uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741 # v3.0.0
+        with: 
+          name: sbom_source_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx
+
+      - name: Download artifact for CI ${{ matrix.name }} image SBOM
+        uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741
+        with: 
+          name: sbom_image_ci_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx
+
+      - name: Download artifact for CI race detection ${{ matrix.name }} image SBOM
+        uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741
+        with: 
+          name: sbom_image_ci_race_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx
+
+      - name: Download artifact for CI Unstripped Binaries ${{ matrix.name }} image SBOM
+        uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741
+        with: 
+          name: sbom_image_ci_unstripped_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx
```

Comment:
Question: why is downloading artifacts necessary, and where are they uploaded from initially? Looking at the workflow I was under the impression these files were generated in the step above 🤔

Reply:
My bad, downloading artifacts is not necessary.